sql/core/src/main/scala/org/apache/spark/sql/execution/Generate.scala

```diff
@@ -34,24 +35,12 @@ import org.apache.spark.sql.catalyst.expressions._
 @DeveloperApi
 case class Generate(
     generator: Generator,
+    output: Seq[Attribute],
```

Comment:
The output of `Generate` is not always identical with `Generator`, e.g. if `join` is `true`.